src/Faker/Provider/es_MX/Person.php

```diff
@@ -0,0 +1,466 @@
+
+
+    /**
+     * @inheritdoc
+     */
+    public function lastName()
+    {
+        return static::randomElement(static::$lastName) . ' ' . static::randomElement(static::$lastName);
+    }
+
+    /**
+     * @example Águila
+     * @return string
+     */
+    public function lastNameMother()
```

Comment:
I don't understand the rationale, `lastNameMother` and `lastNameFather` do the same thing.